clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -905,3 +911,21 @@ def _get_multipart_id(nb_parts: PhilipsNumberOfParts, run_num: str) -> Optional[
         raise ValueError(f"{run_num} is outside of the scope.")
     if nb_parts == PhilipsNumberOfParts.OTHER:
         return None
+
+
+def delete_real_imaginary_files(to: PathLike) -> None:
+    """This function scans the BIDS after it has been written to delete the `_real` and `_imaginary` files.
+
+    Parameters
+    ----------
+    to: PathLike
+        Path to the BIDS
+    """
+
+    import os
+
```

Comment:
Could be nice to add a logging message (INFO ?) for each deleted file (or one for all at once??).
As a user, I would expect the converter to tell me something when deleting files from my system.